components/component-docs.json

```diff
@@ -743,13 +743,6 @@
                     "computed": false
                 }
             },
-            "className": {
```

Comment:
Interestingly - maybe it's been a while we didn't run `build:docs` ? There's a bunch of updates unrelated to this PR.